lib/permutat.g

```diff
@@ -560,10 +560,11 @@ end);
 ##  <Ref Func="MappingPermListList"/> returns the permutation <C>p</C> from the
 ##  previous sentence, i.e.  <A>src</A><C>[</C><M>i</M><C>]^</C><M>p =</M>
 ##  <A>dst</A><C>[</C><M>i</M><C>]</C>.
-##  The permutation <M>\pi</M> fixes all points larger than the maximum of
-##  the entries in <A>src</A> and <A>dst</A>.
+##  The permutation <M>\pi</M> fixes all points which are not contained in
+##  either of <A>src</A> or <A>dst</A>.
```

Comment:
This would be simpler if it was "fixes any point which is not in <A>src</A> or <A>dst</A>"